src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.logoimage {
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  box-sizing: border-box;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  flex-wrap: nowrap;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  height: 60px;
+  line-height: 60px;
+  font-weight: 500;
+}
+
+.menu-item {
+  font-size: 12px;
+  margin: 0 10px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
```

Comment:
Consider using a more accessible color contrast for text color to ensure readability for users with visual impairments. The current color `#000` on a potentially light background might not provide enough contrast.